Doc/reference/datamodel.rst

```diff
@@ -2097,11 +2097,21 @@ Resolving MRO entries
    of classes that will be used instead of the base. The returned tuple may be
    empty: in these cases, the original base is ignored.
 
+
+.. attribute:: type.__orig_bases__
+
+   If the class originally had bases modified by :meth:`~object.__mro_entries__`,
+   this the *bases* tuple.
+
```

Comment:
I'd rather leave documenting this attribute to a separate PR. I don't think it's necessary for now, and I don't have the energy to bikeshed over the precise wording of this right now 😅